scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -157,10 +164,9 @@ sealed trait Execution[+T] extends java.io.Serializable {
    * composition. Every time someone calls this, be very suspect. It is
    * always code smell. Very seldom should you need to wait on a future.
    */
-  def waitFor(conf: Config, mode: Mode): Try[T] = {
+  def waitFor(conf: Config, mode: Mode): Try[T] =
     Try(Await.result(run(conf, mode)(ConcurrentExecutionContext.global),
```

Comment:
Not an issue, but shame Await API doesn't cover returning a try instead of throwing and having to catch it. liftToTry on twitter futures is nice for that